packages/expo-updates/src/Updates.types.ts

```diff
@@ -84,6 +84,20 @@ export type UpdateEvent = {
   message?: string;
 };
 
+// @docsMissing
+/**
+ * An object representing a single log entry from expo-updates logging on the client.
+ */
```

Comment:
```suggestion
// @docsMissing
/**
 * An object representing a single log entry from expo-updates logging on the client.
 */
```

It would be nice to document added object properties. We usually use `@docsMissing` annotation only during the API conversion to the autogenerated doc, when there original code author did not left any comments.

Reply:
Removed `@docsmissing` annotations